<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot1" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>112</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDate|Qt::ImhTime</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCmdESP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>START</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>STOP</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbSendESP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>282</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot3" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>452</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot5" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>282</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot4" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>112</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot6" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>452</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>282</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlLayout_2">
     <item>
      <widget class="QRadioButton" name="radioADC1_2">
       <property name="text">
        <string>ADC1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioADC2_2">
       <property name="text">
        <string>ADC2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccX_2">
       <property name="text">
        <string>Acc X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccY_2">
       <property name="text">
        <string>Acc Y</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccZ_2">
       <property name="text">
        <string>Acc Z</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTsense_2">
       <property name="text">
        <string>T</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>452</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlLayout_3">
     <item>
      <widget class="QRadioButton" name="radioADC1_3">
       <property name="text">
        <string>ADC1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioADC2_3">
       <property name="text">
        <string>ADC2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccX_3">
       <property name="text">
        <string>Acc X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccY_3">
       <property name="text">
        <string>Acc Y</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccZ_3">
       <property name="text">
        <string>Acc Z</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTsense_3">
       <property name="text">
        <string>T</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_3</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlLayout_4">
     <item>
      <widget class="QRadioButton" name="radioADC1_4">
       <property name="text">
        <string>ADC1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioADC2_4">
       <property name="text">
        <string>ADC2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccX_4">
       <property name="text">
        <string>Acc X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccY_4">
       <property name="text">
        <string>Acc Y</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccZ_4">
       <property name="text">
        <string>Acc Z</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTsense_4">
       <property name="text">
        <string>T</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_4</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlLayout_5">
     <item>
      <widget class="QRadioButton" name="radioADC1_5">
       <property name="text">
        <string>ADC1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioADC2_5">
       <property name="text">
        <string>ADC2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccX_5">
       <property name="text">
        <string>Acc X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccY_5">
       <property name="text">
        <string>Acc Y</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccZ_5">
       <property name="text">
        <string>Acc Z</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTsense_5">
       <property name="text">
        <string>T</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_5</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlLayout_6">
     <item>
      <widget class="QRadioButton" name="radioADC1_6">
       <property name="text">
        <string>ADC1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioADC2_6">
       <property name="text">
        <string>ADC2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccX_6">
       <property name="text">
        <string>Acc X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccY_6">
       <property name="text">
        <string>Acc Y</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAccZ_6">
       <property name="text">
        <string>Acc Z</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTsense_6">
       <property name="text">
        <string>T</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_6</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioAccZ">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Acc Z</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioTsense">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>234</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioADC1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>114</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioADC2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>138</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ADC2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioAccY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>186</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Acc Y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioAccX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>162</y>
      <width>89</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Acc X</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>86</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Модуль 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Сдвиг</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineShift">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWidth">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>130</y>
      <width>61</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="layoutColorScale"/>
   </widget>
   <widget class="QPushButton" name="pbChangeShift">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_6"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_5"/>
  <buttongroup name="buttonGroup_4"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
